src/darwin/Framework/CHIP/MTRError.mm

```diff
@@ -252,6 +255,38 @@ + (NSError *)errorForIMStatus:(const chip::app::StatusIB &)status
     case Status::UnsupportedEvent:
         description = NSLocalizedString(@"The event indicated is unsupported on the cluster.", nil);
         break;
+    case Status::PathsExhausted:
+        description = NSLocalizedString(@"The receiver has insufficient resources to support the "
+                                         "specified number of paths in the request.",
```

Comment:
Not sure if this is wording directly from the spec, but "... number of paths specified in the request" would be clearer I think.

Reply:
It's copy/paste from the spec, but I can clean it up like I cleaned up some of the other ones where the spec has grammar or spelling issues....

Reply:
Done.